public/svg/Arrow.tsx

```diff
@@ -0,0 +1,19 @@
+export function Arrow({ onClick }: { onClick: () => void }) {
+  return (
+    <svg
+      width="18"
+      height="18"
+      viewBox="0 0 18 18"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      onClick={onClick}
+    >
+      <path
+        d="M4 7L8.24264 11.2426C8.63316 11.6332 9.26633 11.6332 9.65685 11.2426L13.8995 7"
+        stroke="black"
+        stroke-width="1.5"
+        stroke-linecap="round"
```

Comment:
```suggestion
        strokeWidth="1.5"
        strokeLinecap="round"
```
jsx(tsx)에서는 속성을 스네이크 표기법이 아닌 카멜케이스 표기법을 사용합니다.

Reply:
[🎨 Update: svg 카멜 케이스 표기와 onClick 속성 삭제](https://github.com/GSM-MSG/GAuth-FrontEnd/pull/122/commits/e42fe3f4f1f1eca4acdd473ef3606a6cc507f1e0)